src/Chirp.Web/Pages/UserTimeline.cshtml.cs

```diff
@@ -1,29 +1,24 @@
 using Chirp.Core.Entities;
 using Chirp.Core.Repository;
-using Chirp.Web;
 using Chirp.Web.Models;
-using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace Chirp.Razor.Pages;
+namespace Chirp.Web.Pages;
 
 public class UserTimelineModel : PageModel
 {
     private readonly ICheepService _service;
     private readonly UserManager<Author> _userManager;
     private readonly IAuthorRepository _authorRepository;
-
-
-    public ICollection<CheepViewModel> Cheeps { get; set; }
-    public UserModel UserModel { get; set; }
     
+    public Guid AuthorGuid;
```

Comment:
What's the point of keeping the AuthorGuid when we already have the AuthoDTO(Line: 18)